### DISCUSSION TITLE
Replace "Bitbucket Server" with "Bitbucket Data Center" ?

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#13310] [General] [answered]
I see some occurrences of "Bitbucket Server" in Weblate's docs, and I'm not very familiar with its brands and plans. But looking at Atlassian docs, I see only "Bitbucket Cloud" and "Bitbucket Data Center" in [Bitbucket Rebrand FAQ](https://confluence.atlassian.com/bitbucketserver/bitbucket-rebrand-faq-779298912.html) page, where the latter is (quote) _"previously known as Stash Data Center"_.

Reading the [Bitbucket](https://en.wikipedia.org/wiki/Bitbucket#Bitbucket_Server)'s page at Wikipedia, it says (quote) _"**Bitbucket Server** (formerly known as **Stash** (...)"_.

Should "Bitbucket Server" be replaced with "Bitbucket Data Center" ?

**Comment by nijel** [comment, marked as answer]
It appears that *Server* and *Data Center* were different editions of the same product (https://confluence.atlassian.com/enterprise/bitbucket-server-and-data-center-feature-comparison-953652422.html) and is now gone (https://www.atlassian.com/migration/assess/journey-to-cloud).

https://github.com/WeblateOrg/weblate/pull/13311 renames it.
